### DISCUSSION TITLE
IBAX UYGULAMA YAPMALIMI?

### DISCUSSION THREAD

**Discussion by halildemirellx** [IBAX-io/go-ibax#721] [Ideas] [answered]
### Clear and concise description of the problem

ıbax androıd ve appstore kısmından bir uygulama olmaması

### Suggested solution

appstore ve playstoreda güncel bir uygulama yapılması insanların daha cok gözüne carpacagını dusunmekteyim

### Alternative

uygulamanın youtubeda tanıtılması ve ordada her getirilen referans icin ıbxc almalarını saglanması projenin gelecegını anlatan guzel bir slayt hazırlanıp uygulamanın ilk acılısında konulması gayet guzel bir uygulama olacaktır

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by canrasim** [comment]
BENCE GÜZEL BİR UYGULAMA İLE HERKEZE ULAŞILABİLİR YAPILMALI

**Comment by ibaxio** [comment, marked as answer]
Please refer to this discussion:  #629
